graylog2-web-interface/src/components/event-definitions/event-definition-types/FilterAggregationForm.jsx

```diff
@@ -115,12 +115,6 @@ class FilterAggregationForm extends React.Component {
     this.setState(stateChange);
   };
 
-  handleChange = (event) => {
```

Comment:
@gally47: I had to change this method to make the linter check pass. Could you please double-check if the method is really not used anymore? I don't know our frontend well enough to rule out that there is any magic going on that would rely on this method being present.

Reply:
@mnin already checked this for me 🙏